src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -301,20 +301,21 @@ case class JesBackend(actorSystem: ActorSystem)
       Try(interface.copy(cachedCall.callGcsPath, backendCall.callGcsPath)) match {
         case Failure(ex) =>
           log.error(s"Exception occurred while attempting to copy outputs from ${cachedCall.callGcsPath} to ${backendCall.callGcsPath}", ex)
-          FailedExecutionHandle(ex)
+          FailedExecutionHandle(ex).future
         case Success(_) => postProcess(backendCall) match {
-          case Success(outputs) => SuccessfulExecutionHandle(outputs, backendCall.downloadRcFile.get.stripLineEnd.toInt, backendCall.hash)
+          case Success(outputs) => backendCall.hash map { h =>
+            SuccessfulExecutionHandle(outputs, backendCall.downloadRcFile.get.stripLineEnd.toInt, h, Option(cachedCall)) }
```

Comment:
oh right, I'm still thinking with the old implementation of `backendCall.hash` but even then it doesn't make sense because then it would return a `String`